zerver/webhooks/newrelic/tests.py

```diff
@@ -4,40 +4,123 @@
 class NewRelicHookTests(WebhookTestCase):
     STREAM_NAME = 'newrelic'
     URL_TEMPLATE = "/api/v1/external/newrelic?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'newrelic'
 
-    def test_alert(self) -> None:
-        expected_topic = "Apdex score fell below critical level of 0.90"
-        expected_message = 'Alert opened on [application name]: Apdex score fell below critical level of 0.90 ([view alert](https://rpm.newrelc.com/accounts/[account_id]/applications/[application_id]/incidents/[incident_id])).'
+    def test_open(self) -> None:
+        expected_topic = "New Relic Alert - Test Policy (1234)"
+        expected_message = """
+Incident **opened** for condition: **Server Down** at <time:2020-11-11 22:32:11.146000>
+``` quote
+I am a test Violation
+```
+[View incident](https://alerts.newrelic.com/accounts/2941966/incidents/1234)
+""".strip()
 
         self.check_webhook(
-            "alert",
+            "incident_opened",
             expected_topic,
             expected_message,
-            content_type="application/x-www-form-urlencoded",
+            content_type="application/json",
         )
 
-    def test_deployment(self) -> None:
-        expected_topic = 'Test App deploy'
+    def test_closed(self) -> None:
+        expected_topic = "New Relic Alert - Test Policy (1234)"
         expected_message = """
-**1242** deployed by **Zulip Test**:
+Incident **closed** for condition: **Server Down**
```

Comment:
Do we have the ability to linkify `Incident` here, or is the data not available?

Reply:
Data is available!